examples/units_and_coordinates/spice.py

```diff
@@ -0,0 +1,144 @@
+
+kernel_urls = [
+    "ck/solo_ANC_soc-sc-fof-ck_20180930-21000101_V03.bc",
+    "ck/solo_ANC_soc-stix-ck_20180930-21000101_V03.bc",
+    "ck/solo_ANC_soc-flown-att_20221011T142135-20221012T141817_V01.bc",
+    "fk/solo_ANC_soc-sc-fk_V09.tf",
+    "fk/solo_ANC_soc-sci-fk_V08.tf",
+    "lsk/naif0012.tls",
+    "pck/pck00010.tpc",
+    "sclk/solo_ANC_soc-sclk_20231015_V01.tsc",
+    "spk/de421.bsp",
+    "spk/solo_ANC_soc-orbit-stp_20200210-20301120_280_V1_00288_V01.bsp",
+]
+kernel_urls = [f"http://spiftp.esac.esa.int/data/SPICE/SOLAR-ORBITER/kernels/{url}"
+               for url in kernel_urls]
+
+kernel_files = [cache.download(url) for url in kernel_urls]
+
+###############################################################################
+# Initialize `sunpy.coordinates.spice` with these kernels, which will create
+# classes for `~astropy.coordinates.SkyCoord` to use.
+
+spice.initialize(kernel_files)
+
+###############################################################################
+# We can request the location of the spacecraft in any SPICE frame.  Here, we
+# request it in 'SOLO_HEEQ', which is Stonyhurst heliographic coordinates.
+
+spacecraft = spice.get_body('Solar Orbiter', obstime, spice_frame='SOLO_HEEQ')
+print(spacecraft[:4])
+
+###############################################################################
+# Plot the radial distance from the Sun over the time range.
+
+fig = plt.figure()
+ax = fig.add_subplot()
+ax.plot(obstime.datetime64, spacecraft.distance.to('AU'))
+ax.xaxis.set_major_formatter(DateFormatter('%H:%M'))
+ax.set_xlabel('2022 October 12 (UTC)')
+ax.set_ylabel('Radial distance (AU)')
+ax.set_title('Solar Orbiter distance from Sun center')
+
+###############################################################################
+# We can then transform the coordinate to a different SPICE frame.  When
+# specifying the frame for `~astropy.coordinates.SkyCoord`, SPICE frame names
+# should be prepended with ``'spice_'``.  Here, we transform it to 'SOLO_GAE'.
+
+spacecraft_gae = spacecraft.transform_to("spice_SOLO_GAE")
+print(spacecraft_gae[:4])
+
+###############################################################################
+# Plot the radial distance from the Earth over the time range.
+
+fig = plt.figure()
+ax = fig.add_subplot()
+ax.plot(obstime.datetime64, spacecraft_gae.distance.to('AU'))
+ax.xaxis.set_major_formatter(DateFormatter('%H:%M'))
+ax.set_xlabel('2022 October 12 (UTC)')
+ax.set_ylabel('Radial distance (AU)')
+ax.set_title('Solar Orbiter distance from Earth center')
+
+###############################################################################
+# Finally, we can leverage the Solar Orbiter SPICE kernels to look at
+# instrument pointing.  Let's define a coordinate that points directly along
+# the line of sight of the STIX instrument.
+
+stix_ils = SkyCoord(np.repeat(0*u.deg, len(obstime)),
+                    np.repeat(0*u.deg, len(obstime)),
```

Comment:
Am I right in thinking that with astropy/astropy#15121 we wont need to do this any more?

Reply:
I believe so